### ISSUE TITLE
Bring back Classic UI

### ISSUE THREAD

**Issue by wanglai8** [sereneblue/chameleon#343]
please bring back classic UI so can fast disable chameleon (real profile) or switch to mobile profile,

new UI make more difficult to switch profile

**Comment by sereneblue** [comment]
Hi @wanglai8, 

Can you provide some more details about how you're using Chameleon? I'm open to UX improvements, but I'm not sure what's the best way to go about it. I think the new UI provides some extra functionality (quick overview of settings, toggle Chameleon, fingerprint detection) that is worth the trade off of an extra click to change between profile options. Clicking the shield icon can temporarily disable Chameleon if you'd like to use your real profile.

**Comment by WPFilmmaker** [comment]
I think with one one gets used...I find the new user modern albeit not so fast to navigate but perhaps is because it is a new thing. Also I don't know you but for me chameleon is one of those addons I tweak at the beginning and then rarely touch as opposed to others where you always fiddle it (noscript etc.).

One improvement I would like to see if when you select the UA you see the three buttons for windows linux and make, and once you click they open the menu. I think previously the dropmenu would stay open, now you see three buttons and have to click on them...to me it would make sense having them always open, but depends whether you think they look nicer closed.

**Comment by sereneblue** [comment]
@WPFilmmaker The list of profiles will stay open if one of the options under it is selected. For example, selecting `macOS 10.13 - Firefox 68 ESR` will keep the macOS list open when you see the popup again.

**Comment by WPFilmmaker** [comment]
@sereneblue in my case it is not happening :/ I have random profile (desktop) toggled, change periodically no, and I am using a chrome windows profile but the window menu remains closed (keep in mind i excluded all the other UAs so effectively the 'random profile' can only choose from 1 UA and don't know if it is affects).

**Comment by sereneblue** [comment]
@WPFilmmaker If one of the random X options is selected, it does hide the profile list. This behavior is identical to v0.12.x. You can get the name of the profile being used by looking at the home tab (first tab loaded when the popup opens).

The profile list is only open if you've toggled it by clicking the tab (Windows, etc) or if a specific profile is selected.

**Comment by 0pLuS0** [comment]
@sereneblue 

I want to first say, I truly appreciate all the work and efforrt you put into Chamelon, thank you very much!

I have used Chameleon since you first began the project, and in all honesty, I don't see any benefit to this new UI, to me, it's just a new look is all nothing else.

BUT of course you added in the notifications, seeing numbers now for the things blocked, as well as them being mentioned in the main window of the UI.

To me now, I feel like I am using a Privacy/Adblocking/Tracker application, and Chameleon is suppose to be a spin off of RAS, a ```Spoofer```, and to me, a Spoofer just does that, Spoofs the Browser/OS, and great, of course it blocks all these things, but I don't see that we need to be notified everytime it does it, to me that is just annoying and bloated...

**Comment by sereneblue** [comment]
@0pLuS0 The fingerprint detection panel was removed in v0.20.3 so you won't see those notifications anymore.

**Comment by 0pLuS0** [comment]
@sereneblue Hello!

What about the Popup Notification of ```Numbers``` disaplyed in the Chameleon Icon in Firefox?

THANKS

**Comment by sereneblue** [comment]
@0pLuS0 That's also removed. :)

**Comment by ookangzheng** [comment]
> @sereneblue
> 
> I want to first say, I truly appreciate all the work and efforrt you put into Chamelon, thank you very much!
> 
> I have used Chameleon since you first began the project, and in all honesty, I don't see any benefit to this new UI, to me, it's just a new look is all nothing else.
> 
> BUT of course you added in the notifications, seeing numbers now for the things blocked, as well as them being mentioned in the main window of the UI.
> 
> To me now, I feel like I am using a Privacy/Adblocking/Tracker application, and Chameleon is suppose to be a spin off of RAS, a `Spoofer`, and to me, a Spoofer just does that, Spoofs the Browser/OS, and great, of course it blocks all these things, but I don't see that we need to be notified everytime it does it, to me that is just annoying and bloated...

From what I saw, is a great move for chameleon it self as new version. Entire new UI is good for rebranding it self for now days UI trend. It doesn't make chameleon looks old like from 90's. 

Just my thought, I learnt UI/UX in school and realize that Opensource community is scared to moving forward to new UI or design pattern. It kinda suck for other ppls who is not from CS background. For example: Dnsdist [documentation](https://dnsdist.org/guides/serverselection.html). Full of bloated words and poor design. 

User friendly design: 
1. https://create-react-app.dev/docs/getting-started/
2. https://deno.land/x/docable/docs/
3. https://beakerbrowser.com/docs/

**Comment by 0pLuS0** [comment]
v0.20.8 now is nice, no more Popup numbers of the blocks and the notifications of what it was in the UI.

But the UI is still HUGE, maybe 25%-30% smaller would look better... :)

**Comment by sereneblue** [comment]
@0pLuS0 The older UI was 410x450. v0.20.0 was 510x600. After you brought attention to it, the popup was reduced to 426x502 in v0.20.7. There's not much more that I can comfortably cut without having to make compromises somewhere.

**Comment by 0pLuS0** [comment]
Oh so it's a little smaller now?

THANKS

**Comment by sereneblue** [comment]
@0pLuS0 Yes, it's a bit smaller. You can check this by comparing v0.20.6 to v0.20.8. 

https://addons.mozilla.org/en-US/firefox/addon/chameleon-ext/versions/

**Comment by 0pLuS0** [comment]
Ok, keep up the great work!

THANKS
